composer.json

```diff
@@ -6,7 +6,7 @@
     "keywords": [
         "sonata",
         "formatter",
-        "editor"
+        "ckeditor"
```

Comment:
I would add ckeditor, and not remove the editor keyword, since this allows multiple kinds of editors